MARIE_AND_BERT/readme.md

```diff
@@ -0,0 +1,127 @@
+# Marie and BERT (Marie 3.0)
+
+The ``Marie and Bert`` a.k.a `Marie 3.0` project is developed by [Xiaochi Zhou](xz378@cam.ac.uk)
+and [Shaocong Zhang](sz375@cam.ac.uk) and [Mehal Agarwal](ma988@cam.ac.uk).
+
+A demonstration webpage is deployed at [Marie Website](http://159.223.42.53:5003/). For any issue on the 
+website, please contact [Xiaochi Zhou](xz378@cam.ac.uk) (Before 01.08.2023).
+
+The contact information after 08.2023 will soon be updated. 
```

Comment:
There is almost no point in this bit - it will be obsolete in about one month's time, and I can promise you no-one will update it. Therefore I recommend removing this.